File: src/main/java/com/example/store/data/Authority.java
package com.example.store.data;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Data
@Table(name = "authority")
@AllArgsConstructor
@NoArgsConstructor
public class Authority {
    @Id
    private String name;
}

File: src/main/kotlin/com/example/store/data/Authority.kt
package com.example.store.data

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "authority")
class Authority() {
    @Id
    private var name: String? = null

    fun getName(): String? {
        return name
    }
    fun setName(name: String) {
        this.name = name
    }
}